src/Avalonia.Themes.Fluent/Controls/Expander.xaml

```diff
@@ -1,141 +1,178 @@
+    <SolidColorBrush x:Key="ExpanderDropDownBorderBrush" Color="{DynamicResource SystemBaseLowColor}"/>
+
+    <SolidColorBrush x:Key="ExpanderForeground" Color="{DynamicResource SystemBaseHighColor}"/>
+
+    <SolidColorBrush x:Key="ExpanderChevronForeground" Color="{DynamicResource SystemBaseHighColor}"/>
+  </Styles.Resources>
   <Style Selector="Expander">
-    <Setter Property="ContentTransition">
-      <Setter.Value>
-        <CrossFade Duration="00:00:00.25" />
-      </Setter.Value>
-    </Setter>
+    <Setter Property="Background" Value="{DynamicResource ExpanderBackground}" />
+    <Setter Property="BorderThickness" Value="{DynamicResource ExpanderBorderThickness}" />
+    <Setter Property="BorderBrush" Value="{DynamicResource ExpanderBorderBrush}" />
+    <Setter Property="Padding" Value="{DynamicResource ExpanderHeaderPadding}" />
+    <Setter Property="HorizontalAlignment" Value="Stretch" />
+    <Setter Property="HorizontalContentAlignment" Value="Left" />
+    <Setter Property="VerticalContentAlignment" Value="Center" />
   </Style>
-  <Style Selector="Expander[ExpandDirection=Down]">
+  <Style Selector="Expander">
     <Setter Property="Template">
       <ControlTemplate>
-        <Border Background="{TemplateBinding Background}">
-          <Grid RowDefinitions="Auto,*">
-            <ToggleButton Name="PART_toggle" Grid.Row="0"  Content="{TemplateBinding Header}" IsChecked="{TemplateBinding IsExpanded, Mode=TwoWay}" />
-            <ContentPresenter Name="PART_ContentPresenter"
-                              Grid.Row="1"
-                              IsVisible="{TemplateBinding IsExpanded}"
-                              ContentTemplate="{TemplateBinding ContentTemplate}"
-                              Content="{TemplateBinding Content}"
-                              HorizontalContentAlignment="{TemplateBinding HorizontalContentAlignment}"
-                              VerticalContentAlignment="{TemplateBinding VerticalContentAlignment}"
-                              Padding="{TemplateBinding Padding}" />
-          </Grid>
-        </Border>
+        <DockPanel>
+            <ToggleButton x:Name="ExpanderHeader"
+                          Background="{TemplateBinding Background}"
+                          BorderBrush="{TemplateBinding BorderBrush}"
+                          BorderThickness="{TemplateBinding BorderThickness}"
+                          IsEnabled="{TemplateBinding IsEnabled}"
+                          Padding="{TemplateBinding Padding}"
+                          Content="{TemplateBinding Header}"
+                          ContentTemplate="{TemplateBinding HeaderTemplate}"
+                          HorizontalAlignment="{TemplateBinding HorizontalAlignment}"
+                          IsChecked="{Binding Path=IsExpanded, Mode=TwoWay, RelativeSource={RelativeSource TemplatedParent}}" />
+            <Border x:Name="ExpanderContent"
```

Comment:
It's preferred to use "PART_" naming in default styles. Here and above.

Reply:
I thought the "PART_" prefix was only for elements which are accessed from the control's code, such as in the `OnApplyTemplate` function...have I been getting that wrong this whole time?

Reply:
For UWP/WPF: "PART_ExpanderContent" should only be used if the control is referenced in code-behind. Standard naming convention ("ExpanderContent") should be used for controls used by name in the template/style only. Not sure if Avalonia convention is different for their styling system.

Reply:
Well, as for me there are no a lot of difference between controls used in OnApplyTemplate and controls that can be potentially used there in some day or just additional components of layout. Especially when they all can be used in the styles without OnApplyTemplate. And personally I used "PART_" everywhere for consistency,

On other hand, as I mentioned above, it's not a blocker for me.

Reply:
Well, the "PART_" naming simply means it is a part of the template referenced or required by code. A style/template may have to name any number of controls for styling reasons but most don't need to be referenced outside the style itself. The "PART_" is to indicate a link from code -> style NOT from style -> style (which is arbitrary and the code doesn't need to know).

Reply:
If it's not a blocker, then I think I'll continue to reserve "PART_" for the bits used in code, as I previously have (and as robloo noted is standard practice in WPF and UWP). Working on the corners as we speak.